### ISSUE TITLE
Screen reader users need to go through unnecessary elements between search field and search results

### ISSUE THREAD

**Issue by juhakujala** [City-of-Helsinki/kerrokantasi-ui#706]
Here is the current situation of the search view:
<img width="1054" alt="Screenshot 2019-06-19 at 13 26 58" src="https://user-images.githubusercontent.com/7901266/59758333-5a585980-9296-11e9-8773-459be15afb59.png">

Problem:
There's a lot of elements between the search input field and actual search results.

Suggestion:
Add a link for screen readers to jump from the search field to the results. The jump link should be placed straight after the search input element. We could also provide a link back to the search field from the search results.
